src/crypto/x509/x509.go

```diff
@@ -1761,6 +1765,15 @@ func buildExtensions(template *Certificate, subjectIsEmpty bool, authorityKeyId
 		n++
 	}
 
+	if template.KeyUsage == KeyUsageDigitalSignature && template.IsDC && !oidInExtensions(oidExtensionDelegatedCredential, template.ExtraExtensions) {
```

Comment:
Do we need to do anything to mark this extension as non-critical?

Reply:
The return strut `ret` is:

```
type Extension struct {
	Id       asn1.ObjectIdentifier
	Critical bool `asn1:"optional"`
	Value    []byte
}
```

by default, if not set to true, is marking it false.

Will that suffice?

Reply:
Yep! It might be good to make that clear in a comment.

Reply:
Ok!